bodhi/server/models.py

```diff
@@ -1506,6 +1506,21 @@ def get_testcase_karma(self, testcase):
                         bad += 1
         return bad * -1, good
 
+    def beautify_title(self, amp=False):
+        if len(self.builds) == 1:
+            return packagename_from_nvr(self, self.builds[0].nvr)
+        elif len(self.builds) == 2:
+            return " and ".join([packagename_from_nvr(self, build.nvr) for build in self.builds[:2]])
```

Comment:
Since you know that ```len(builds) == 2```, you can probably leave off the ```:2``` and get the line short enough to make ```flake8``` happy.